core/domain/wipeout_service.py

```diff
@@ -226,7 +226,8 @@ def delete_users_pending_to_be_deleted():
         )
 
     email_subject = 'User Deletion job result'
-    email_manager.send_mail_to_admin(email_subject, email_message)
+    if not feconf.CAN_SEND_EMAILS:
```

Comment:
Is this if-check backwards? Shouldn't we only send mail if we can?

Also, could you add tests for each of these cases to demonstrate that mail is, and is not, sent based on whether CAN_SEND_EMAILS is set?

Reply:
Yeah, mistake, done.